Task2.java

```diff
@@ -0,0 +1,19 @@
+public class Task2 {
+    public static void main(String[] args) {
+    int a = 6;
+    int b = 4;
+    int Sum = a+b;
```

Comment:
variables have to start from lowercase letter according to ''Naming Conventions''

Reply:
fixed